content/posts/en/eth2-staking-with-k8s-prysm.md

```diff
@@ -355,43 +366,44 @@ On the machine you plan to run NFS:
     sudo exportfs -a
     ```
 
-On your controller and worker nodes, enable NFS support by installing nfs-common:
+
+On your master and worker nodes, enable NFS support by installing nfs-common:
 
 ```bash
 sudo apt install nfs-common
 ```
 
 ### Prepare Validator Wallets
 
-Please refer to the Prysm’s [official documentation](https://docs.prylabs.network/docs/mainnet/joining-eth2/#step-4-import-your-validator-accounts-into-prysm).
+Please refer to Prysm’s [official documentation](https://docs.prylabs.network/docs/mainnet/joining-eth2/#step-4-import-your-validator-accounts-into-prysm).
 
-Let’s get back to the NFS server. We need to configure the wallet directories that we created in the previous section. Before proceeding, please have your validator keys placed on your NFS machine. We use `$HOME/eth2.0-deposit-cli/validator_keys` as the example path to the validator keys. To create a Prysm’s wallet and import your validator keys into it, we use the Prysm startup script.
+Let’s get back to the NFS server. We need to configure the wallet directories that we created in the previous section. Before proceeding, please have your validator keys placed on your NFS machine. We use `$HOME/eth2.0-deposit-cli/validator_keys` as the example path to the validator keys. To create a wallet and import your validator keys for Prysm validator client, we use Prysm’s startup script.
 
 1. Please follow Prysm’s [documentation](https://docs.prylabs.network/docs/install/install-with-script/#downloading-the-prysm-startup-script) to download Prysm startup script.
 
-2. Execute the startup script. We have put the validator keys under `$HOME/eth2.0-deposit-cli/validator_keys`which is the key directory we are using in the following command.
+2. Execute the startup script. We placed the validator keys under `$HOME/eth2.0-deposit-cli/validator_keys`which is the key directory we are using in the following command.
```

Comment:
How about " Execute the startup script with `--keys-dir=<path/to/validator-keys>`"? Just hint the reader to replace the string.

Reply:
Sounds good! So it can be like this: 

2. Execute the startup script with `--keys-dir=<path/to/validator-keys>` (Remember to replace it with the directory you place the keys).  We placed the validator keys under `$HOME/eth2.0-deposit-cli/validator_keys`.

```bash
sudo ./prysm.sh validator accounts import --keys-dir=<path/to/validator-keys>
```